apps/src/sites/code.org/pages/public/yourschool.js

```diff
@@ -37,9 +37,6 @@ function showYourSchool() {
         hideMap={yourschoolElement.data('parameters-hide-map')}
         prefillData={prefillData}
         currentCensusYear={yourschoolElement.data('parameters-school-year')}
-        showProfessionalLearningBanner={yourschoolElement.data(
```

Comment:
I believe this parameters are coming from `yourschool.haml` here: https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/yourschool.haml#L28-L29, if we're no longer using them, let's clean this up as well.

Reply:
It looks like teacher_application_mode is used in the census_announcement, but I removed the other line!